vlad_task4/install-opencv.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+cwd=$(pwd)
```

Comment:
generally, try avoiding building stuff if a standard ubuntu package is available...
https://linuxize.com/post/how-to-install-opencv-on-ubuntu-20-04/